app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -413,7 +438,10 @@ protected void onResume() {
         //Otherwise this will get triggered by the broadcastreceiver in the onCreate
         AppProvider appProvider = KissApplication.getDataHandler(this).getAppProvider();
         if (appProvider != null && appProvider.isLoaded())
-            checkShowFavoritesBar(searchEditText.getText().toString().length() > 0);
+            // Favorites needs to be displayed again if the quickfavorite bar is active,
+            // Not sure why exactly, but without the "true" the favorites drawable will disappear
```

Comment:
This is concerning me, since I don't understand the why.

I've optimized the display of favorites compared to the actual implementation, but I don't get why the drawable would disappear. I'm assuming this is related to some optimizations in the manifest about `stateNotNeeded` or similar (no this one, since I tried without it and it's still failing) but I can't get it to work properly.